internal/dockercompose/client.go

```diff
@@ -39,7 +39,6 @@ var dcProjectOptions = []compose.ProjectOptionsFn{
 	compose.WithResolvedPaths(true),
 	compose.WithNormalization(true),
 	compose.WithOsEnv,
-	compose.WithDotEnv,
```

Comment:
mostly for my curiosity - why did this change? since you seem to add it down below

Reply:
The `WithDotEnv` function is what actually applies and reads the file and loads it into the `Environment` option as key-value pairs. The default filename is `.env` in the current directory. `WithEnvFile` changes the filename, but the `WithDotEnv` needs to follow specifying the filename in the slice of option functions to read it.

Reply:
See
- https://github.com/compose-spec/compose-go/blob/master/cli/options.go#L175
- https://github.com/compose-spec/compose-go/blob/master/cli/options.go#L183